package.json

```diff
@@ -41,7 +41,7 @@
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "husky": "^8.0.3",
-    "lint-staged": "^15.1.0",
+    "lint-staged": "^15.2.0",
```

Comment:
Jest jakieś uzasadnienie zmiany wersji tej paczki czy to przypadkowa zmiana?

Reply:
Przypadkowa zmiana i guess?